pgmpy/tests/test_inference/test_ExactInference.py

```diff
@@ -47,6 +47,18 @@ def setUp(self):
     # All the values that are used for comparison in the all the tests are
     # found using SAMIAM (assuming that it is correct ;))
 
+    def test_query_raises_for_empty_variables(self):
+        model = DiscreteBayesianNetwork([("A", "B")])
```

Comment:
An easier way to create a model is to import an existing one. It will also simplify the code. For example:
```
from pgmpy.utils import get_example_modele
model = get_example('earthquake')
```